src/payment/data/sagas.test.js

```diff
@@ -41,14 +41,15 @@ import '../__factories__/basket.factory';
 
 import * as cybersourceService from '../payment-methods/cybersource';
 import {
-  DEFAULT_PAYMENT_STATE_POLLING_DELAY_SECS,
   DEFAULT_PAYMENT_STATE_POLLING_MAX_ERRORS,
   PAYMENT_STATE,
-  POLLING_PAYMENT_STATES,
+  POLLING_PAYMENT_STATES, WAFFLE_FLAGS,
```

Comment:
fix for this formatting issue is already complete and will be pushed with other preliminary feedback.